src/python/grpcio/grpc/_server.py

```diff
@@ -278,6 +278,11 @@ def set_trailing_metadata(self, trailing_metadata):
             self._state.trailing_metadata = trailing_metadata
 
     def abort(self, code, details):
+        # treat OK like other invalid arguments: fail the RPC
+        if code == grpc.StatusCode.OK:
+            logging.warning(
+                'abort() called with Status.Code.OK; returning UNKNOWN')
```

Comment:
s/Status.Code.OK/StatusCode.OK

Reply:
Also, I think we should reset the details to empty string, and not keep the details passed to us with UNKNOWN.

Reply:
Oops nice typo catch. I assigned `details = ''` in this case. This will necessitate changing the unit tests to match.